manual.typ

```diff
@@ -593,10 +588,13 @@ merge-path({
 
 == Groups <groups>
 Groups allow scoping context changes such as setting stroke-style, fill and transformations.
-```typ
-#group(content, name: none)
+
+```typc
+group(content, name: none, anchor: none)
 ```
 
+Note: You can pass `content` a function of the form `ctx => array` that returns the groups children. This way you get access to the groups context object.
```

Comment:
I don't think having the return type `array` is very clear as we don't explain anywhere that actually the draw functions return arrays that implicitly get joined. I also think we should rename `content` to `body` as you can't use Typst's content. Maybe just say "`body` can also be function takes a single parameter, a dictionary that describes the group's current context such as styling".

We should also start making some separate developer documentation for the context dictionary but I'm not to bothered about it right now.

Reply:
Yeah, but we should first generate function docs from the code doc. I do not like having both.